contracts/flattened/Registry.sol.flattened

```diff
@@ -600,7 +607,7 @@ contract Registry is Ownable, FundsRecovery {
 
     function transferCollectedFeeTo(address _beneficiary) public onlyOwner{
         uint256 _collectedFee = token.balanceOf(address(this));
-        require(_collectedFee > 0);
+        require(_collectedFee > 0, "collected fee cannot be less than zero");
```

Comment:
```suggestion
        require(_collectedFee > 0, "collected fee must be greater than zero");
```

Reply:
It's autogenerated file. Changes can't be made here directly.